projects/ngneat/reactive-forms/src/lib/formGroup.ts

```diff
@@ -269,4 +271,8 @@ export class FormGroup<T extends Obj = any, E extends object = any> extends NgFo
   setDisable(disable = true, opts?: ControlEventOptions) {
     disableControl(this, disable, opts);
   }
+
+  persist(key: string, storage: PersistManager<T> = new LocalStorageManager<T>()) {
```

Comment:
The idea is to save it automatically upon any change by subscribing to `valueChanges` with a configurable `debounceTime`

Reply:
Ok, got you. Should the persist function return the observable or it should it be kept within the form group? I would return it in order that it can be properly unsubscribed.

Another question: Any suggestion when to restore the value from the storage?

Reply:
1. Yes, it should return the subscription 
2. In the function call.

Reply:
updated the PR